doc/getting_started/index.rst

```diff
@@ -159,6 +159,12 @@ Set Up a Toolchain
    setting :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` appropriately for a non-SDK
    toolchain.
 
+   As another way to skip setting up a toolchain, you may be able for
+   some configurations (e.g.: non-MCU x86 and x86_64 boards on Linux) to
+   re-use the development tools provided by your operating system that
+   `you already installed in the first step <host_setup_>`_. To try this
```

Comment:
It's not a valid assumption that all host environments have toolchains installed after following the above steps. That's only true on Linux -- neither the macOS nor the Windows setup guide installs a host toolchain.

So I'd suggest deleting this part.

Reply:
I guess you mean dropping the "already installed... " link? github highlights the whole commit for some reason.

I put the "you _may_ be able for _some_ ... (e.g: _Linux_)" caveat at the start and added this backward reference to cater for the overly optimistic and impatient Linux user who skipped even the very first OS setup but I think you're right, this is assuming too little, will drop the link.

Reply:
(github highlights a different number of lines in the "conversation" tab compared to the "Files changed" tab, weird)